scripts/apps/authoring/attachments/actions.ts

```diff
@@ -35,7 +36,13 @@ export function selectFiles(files: Array<File>) {
                 .map((file) => file.name);
 
             if (filenames.length) {
-                notify.error(gettext('Sorry, but files "{{ filenames }}" are too big.', {filenames: filenames.join(', ')}));
+                notify.error(gettext(
+                    'Sorry, but some files "{{ filenames }}" are bigger than limit ({{ limit }}).',
```

Comment:
I think it's better to remove the spaces between double curly brackets and variable names so it's more clear for the translators that it's a placeholder and must not be translated.